gravitee-apim-cypress/cypress/integration/rest-api/apis/mock-policy-tests.ts

```diff
@@ -70,16 +71,14 @@ context('Create a mock policy on a API v1 (path based)', () => {
     });
   });
 
-  it('should successfully call the mocked API endpoint (v1 API)', function () {
-    cy.retryRequest(`${Cypress.env('gatewayServer')}${createdApi.context_path}`, {
-      retries: 15,
-      function: function (response) {
-        if (response.status !== 200) {
-          throw new Error(`still wrong status`);
-        }
+  it('should successfully call the mocked API endpoint', function () {
+    requestGateway(
+      {
+        method: 'GET',
+        url: `${Cypress.env('gatewayServer')}${createdApi.context_path}`,
       },
-      timeout: 1000,
-    }).should((response: Cypress.Response<any>) => {
+      {},
```

Comment:
So, if I don’t want to provide any request options I still need to pass an empty object?

And I don't know, would it maybe make sense to make 'GET' the default method? For the original Cypress request, for instance, you could just write `cy.request(url)` to perform call the `url` with a GET.